### PR TITLE
Feat/create home page

### PR DESCRIPTION
**Pull request by ArmanKhoyetsyan**
create home page and constantly-page

### CODE REVIEW

File: dwin_quiz/src/App.js [hunk 1]
```diff
@@ -1,23 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router';
+import './App.scss';
+import ConstantlyPage from './component/constantly-page/Component';
+import LOgin from './LOgin';
```
Comment (M7Arman) [inline dwin_quiz/src/App.js:4]: 1. only the first letter should be uppercase
2. why login component is in src/ folder? it should be inside pages/ or components/

File: dwin_quiz/src/App.js [hunk 2]
```diff
@@ -1,23 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router';
+import './App.scss';
+import ConstantlyPage from './component/constantly-page/Component';
```
Comment (M7Arman) [inline dwin_quiz/src/App.js]: why the file name is Component?

File: dwin_quiz/src/component/constantly-page/menu-block/Menu-block.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import { useCallback } from 'react';
+
+export default function MenuBlock() {
+    const navigate = useNavigate();
+     return (
+        <div className='menuBlock'>
+            <div>
+                <div className='dashBoardBlock'>
+                    <button
+                        style={{ cursor: 'pointer', }}
```
Comment (M7Arman) [inline dwin_quiz/src/component/constantly-page/menu-block/Menu-block.js]: move style to css file

File: dwin_quiz/src/component/constantly-page/menu-block/Menu-block.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                        onClick={useCallback(() => navigate('/home'), [navigate])}>
+                        <ImgDashBoard className='imgDashBoard' />
+                        DashBoard
+                    </button>
+                </div>
+                <div className='supportBlock'>
+                    <button style={{ cursor: 'pointer', }}
```
Comment (M7Arman) [inline dwin_quiz/src/component/constantly-page/menu-block/Menu-block.js]: move style to css file

**Review by M7Arman — APPROVED** [review]

File: dwin_quiz/src/App.js (hunk 1 above)
Comment (ArmanKhoyetsyan) [reply dwin_quiz/src/App.js:4]: This is how I created it just to see if routing works or not:And then I'll remove that and add the Suren component instead

**Pull request merged**